graph-service/analyze.py

```
@@ -0,0 +1,115 @@
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s")
+logger = logging.getLogger("analytics")
+
+NEO4J_HOST = os.getenv("NEO4J_HOST", "neo4j")
+NEO4J_USER = os.getenv("NEO4J_USER", "neo4j")
+NEO4J_PASSWORD = os.getenv("NEO4J_PASSWORD", "password")
+NEO4J_URI = f"bolt://{NEO4J_HOST}:7687"
+
+GDS_GRAPH_NAME = "org_graph"
+
+
+def run_analytics():
```

Ruff (0.11.9)

`run_analytics` is too complex (11 > 10)

(C901)